src/components/main/RowMenu.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { Dispatch, SetStateAction, useState } from 'react';
+
+import FolderIcon from '@mui/icons-material/Folder';
+import KeyboardArrowRightIcon from '@mui/icons-material/KeyboardArrowRight';
+import { Box, Button, IconButton } from '@mui/material';
+
+import { DiscriminatedItem, ItemType } from '@graasp/sdk';
+
+import { useBuilderTranslation } from '@/config/i18n';
+import {
+  TREE_MODAL_MY_ITEMS_ID,
+  buildItemRowArrowId,
+} from '@/config/selectors';
+import { BUILDER } from '@/langs/constants';
+
+interface MenuRowProps {
+  ele: DiscriminatedItem;
+  fetchSubItems: () => void;
+  selectedId: string;
+  setSelectedId: Dispatch<SetStateAction<string>>;
+  itemIds: string[];
+}
+
+const isTreeItemDisabled = ({
+  itemPath,
+  itemIds,
+}: {
+  itemPath: string;
+  itemIds: string[];
+}) => {
+  if (itemPath && itemIds.length) {
+    // eslint-disable-next-line no-plusplus
+    for (let i = 0; i < itemIds.length; i++) {
+      const pathsSerious = itemPath
+        ?.split('.')
+        .map((ele) => ele.replaceAll('_', '-'));
+
+      const isItemChild = pathsSerious.indexOf(itemIds[i]);
+      if (isItemChild > -1) {
+        return isItemChild > -1;
+      }
+    }
+  }
+  return false;
```

Comment:
You're disabling the option if the folder is a parent right? 
```suggestion
const itemPaths = itemIds.map((ele) => ele.replaceAll('_', '-'));
return itemPaths.some(item => item.path.includes(itemPath))
```
Be careful not to use `?.` everywhere, here you assume `itemPath` exist by typing it.